apps/src/templates/teacherDashboard/AssignmentVersionSelector.jsx

```diff
@@ -117,7 +164,7 @@ export default class AssignmentVersionSelector extends Component {
         <PopUpMenu
           isOpen={this.state.isMenuOpen}
           targetPoint={this.state.targetPoint}
-          offset={{x: 0, y: 0}}
+          offset={menuOffset}
```

Comment:
Can you help me understand why we are using the offset for this purpose rather than setting styles that are right-aligned? I don't have a great visual of how this fits into the document layout, but naively, I'm nervous about right-aligning something by adding left padding when it could be right-aligned via other styles instead.

Reply:
yes! by default, we left-align the dropdown content like this:
<img width="447" alt="Screen Shot 2019-04-17 at 3 23 39 PM" src="https://user-images.githubusercontent.com/9812299/56324802-cb5b8380-6124-11e9-8bdb-c5d09f160551.png">

but because the version dropdown sits on the right-hand side of the script overview page, the dropdown contents would be cut-off if we left the default. to mitigate this, i'm using the width of the `select` element to move the dropdown content to be right aligned instead:
<img width="458" alt="Screen Shot 2019-04-17 at 3 25 49 PM" src="https://user-images.githubusercontent.com/9812299/56324895-183f5a00-6125-11e9-96ad-6ca4e4561f2e.png">

Reply:
💡 makes perfect sense now, thank you!